### PR TITLE
Task/bws163 get objects endpoint

### PR DESCRIPTION
**Pull request by Donymak**
image

### CODE REVIEW

File: backend/src/common/types/types.ts [hunk 1]
```diff
@@ -89,6 +89,10 @@ export { type ObjectUploadRequestDto } from './object-upload/object-upload';
...
   type BSObjectUploadParamsDto,
   type BSObjectDownloadParamsDto,
-} from './bs-object/bs-object-params';
-export { type UploadPayload } from './bs-object/bs-object';
+  type BSObjectGetRequestParamsDto,
+  type BSObjectGetFilter,
+  type BSObjectGetResponseItemDto,
+  type BSObjectGetResponseDto,
+} from 'src/common/types/bs-object/bs-objec';
+export { type UploadPayload } from 'src/common/types/bs-object/bs-object-upload-payload.type';
```
Comment (what1s1ove) [inline backend/src/common/types/types.ts]: move it back

**Review by what1s1ove — CHANGES_REQUESTED** [review]
👍

File: backend/src/common/types/types.ts (hunk 1 above)
Comment (Donymak) [reply backend/src/common/types/types.ts]: Where?)
Upload payload was in file `'src/common/types/bs-object/bs-object'`
As in other folders with types, a file with the name of an entity(space, worker, etc.) of the exact folder contains reexports from the built shared package. File with name bs-object is used as in other folders, code style is OK(I guess)

Here I see only a typo on 96'st string)

Comment (what1s1ove) [reply backend/src/common/types/types.ts]: image

**Review by Yaroslaww-1 — DISMISSED** [review]

**Review by ania-leichenko — DISMISSED** [review]

File: backend/src/api/bs/bs.api.ts [hunk 2]
```diff
@@ -137,6 +138,29 @@ const initBsApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
...
+      const [, token] = req.headers?.authorization?.split(' ') ?? [];
+
+      const spaces = await bsObjectService.getObjects({
+        spaceId: req.params.id,
+        from: req.query.from,
+        count: req.query.count,
+        token,
+      });
+
+      return rep.send(spaces).status(HttpCode.OK);
```
Comment (junojei) [inline backend/src/api/bs/bs.api.ts]: maybe better name objects or spaceObjects?
```suggestion
      const objects = await bsObjectService.getObjects({
        spaceId: req.params.id,
        from: req.query.from,
        count: req.query.count,
        token,
      });

      return rep.send(objects).status(HttpCode.OK);
```

File: backend/src/common/types/types.ts [hunk 3]
```diff
@@ -90,6 +90,10 @@ export { type ObjectUploadRequestDto } from './object-upload/object-upload';
 export {
   type BSObjectUploadParamsDto,
   type BSObjectDownloadParamsDto,
-} from './bs-object/bs-object-params';
-export { type UploadPayload } from './bs-object/bs-object';
+  type BSObjectGetRequestParamsDto,
+  type BSObjectGetFilter,
+  type BSObjectGetResponseItemDto,
+  type BSObjectGetResponseDto,
+} from '~/common/types/bs-object/bs-object';
```
Comment (junojei) [inline backend/src/common/types/types.ts]:
```suggestion
} from './bs-object/bs-object';
```

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**